README.md

```diff
@@ -5,77 +5,75 @@
 
 JupyterLab extension to notify cell completion
 
-![notify-extension-in-action](https://github.com/deshaw/jupyterlab-notify/blob/main/docs/notify-screenshot.png?raw=true)
-
-This is inspired by the notebook version [here](https://github.com/ShopRunner/jupyter-notify).
```

Comment:
Let's keep it as "The initial version of this extension was inspired by ...", probably at the bottom of README.